web_server/posts/models.py

```diff
@@ -74,16 +76,21 @@ class Post(models.Model):
         (PRIVATE, "Private"),
         (SERVERONLY, "Server Admins Only")
     )
-    visibility = models.CharField(max_length=16, choices=VISIBILITY_CHOICES, default=FRIENDS)
+    visibility = models.CharField(
+        max_length=16, choices=VISIBILITY_CHOICES, default=FRIENDS)
 
     # List of user URI's who can read this message
-    visibleTo = models.ManyToManyField(Author, related_name="posts_granted_access_to", blank=True)
+    visibleTo = models.ManyToManyField(
+        Author, related_name="posts_granted_access_to", blank=True)
 
     # Unlisted posts are hidden from users. By default posts should show to users.
     unlisted = models.BooleanField(default=False)
 
     def __str__(self):
-        post_snippet_length = 15 # number of chars to show in content snippet before cutting off with elipsis
-        result = f'{self.visibility} post by {self.author}: '
-        result += f'"{self.content[:post_snippet_length]}{"..." if len(self.content) >= post_snippet_length else ""}"'
-        return result
\ No newline at end of file
+        # number of chars to show in content snippet before cutting off with elipsis
+        post_snippet_length = 15
+        result = "{} post by {}: ".format(self.visibility, self.author)
```

Comment:
Same issue with f string and formatting here. I notice additional line breaks are being added as well. Are my lines too long?